File: src/main/java/net/ndrei/teslacorelib/containers/IContainerSlotsProvider.java
package net.ndrei.teslacorelib.containers;

import net.minecraft.inventory.Slot;

import java.util.List;

/**
 * Created by CF on 2016-12-21.
 */
public interface IContainerSlotsProvider {
    List<Slot> getSlots(BasicTeslaContainer container);
}

File: src/main/kotlin/net/ndrei/teslacorelib/containers/IContainerSlotsProvider.kt
package net.ndrei.teslacorelib.containers

import net.minecraft.inventory.Slot

/**
 * Created by CF on 2017-06-28.
 */
interface IContainerSlotsProvider {
    fun getSlots(container: BasicTeslaContainer<*>): MutableList<Slot>
}
